src/nyx/features/intensity.cpp

```diff
@@ -66,29 +62,17 @@ void PixelIntensityFeatures::calculate(LR& r)
 	val_ROOT_MEAN_SQUARED = sqrt(val_ENERGY / n);
 	val_INTEGRATED_INTENSITY = integInten;
 
-	// --Centroid and Compactness
-	//val_CENTROID_X = cen_x;
-	//val_CENTROID_Y = cen_y;
-	//
-	//Moments2 mom2;
-	//for (auto& px : lr.raw_pixels)
-	//{
-	//	double dst = std::sqrt(px.sqdist(cen_x, cen_y));
-	//	mom2.add(dst);
-	//}
-	//double compa = mom2.std() / n;
-	//val_COMPACTNESS = compa;
-
 	// --MAD, VARIANCE, STDDEV
 	double mad = 0.0,
 		var = 0.0;
 	for (auto& px : r.raw_pixels)
 	{
-		mad += std::abs(px.inten - mean_);
-		var += (px.inten - mean_) * (px.inten - mean_);
+		double diff = px.inten - mean_;
+		mad += std::abs(diff);
+		var += diff * diff;
 	}
 	val_MEAN_ABSOLUTE_DEVIATION = mad / n;
-	var /= n;
+	var /= (n-1);

```

Comment:
aux_area is initialized to 0 for LR instances. 
should we have some check for division by zero?

Reply:
Check added. Thanks!